plugins/flowdock/app/views/samson_flowdock/_notify_buddy_box.html.erb

```diff
@@ -1,29 +1,61 @@
 <% if @deploy.waiting_for_buddy? && @deploy.stage.send_flowdock_notifications? %>
-  <buddy-request-box default-buddy-request-message="<%= default_flowdock_message(@deploy) %>"
-                     flowdock-flows="<%= @deploy.stage.enabled_flows_names.join(', ') %>"
-                     deploy="<%= @deploy.id %>" ></buddy-request-box>
-
-  <script type="text/ng-template" id="templates/buddy_request_box">
-    <div ng-show="shouldDisplayFeedback()">
-      <div class="alert" ng-class="successful ? 'alert-success' : 'alert-danger'">
-        <span>{{ message }}</span>
-      </div>
-    </div>
+  <%= form_tag flowdock_notify_path(deploy_id: @deploy.id+1000) do %>
```

Comment:
good catch ... that was me trying to trigger an error ...

On Tue, Jun 14, 2016 at 1:08 AM, Zehua Liu <notifications@github.com> wrote:

> In plugins/flowdock/app/views/samson_flowdock/_notify_buddy_box.html.erb
> <https://github.com/zendesk/samson/pull/1064#discussion_r66925255>:
>
> > @@ -1,29 +1,61 @@
> >  <% if @deploy.waiting_for_buddy? && @deploy.stage.send_flowdock_notifications? %>
> > -  <buddy-request-box default-buddy-request-message="<%= default_flowdock_message(@deploy) %>"
> > -                     flowdock-flows="<%= @deploy.stage.enabled_flows_names.join(', ') %>"
> > -                     deploy="<%= @deploy.id %>" ></buddy-request-box>
> > -
> > -  <script type="text/ng-template" id="templates/buddy_request_box">
> > -    <div ng-show="shouldDisplayFeedback()">
> > -      <div class="alert" ng-class="successful ? 'alert-success' : 'alert-danger'">
> > -        <span>{{ message }}</span>
> > -      </div>
> > -    </div>
> > +  <%= form_tag flowdock_notify_path(deploy_id: @deploy.id+1000) do %>
>
> Why +1000?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/zendesk/samson/pull/1064/files/bafc898b979eb34b71dcc115de75abb0e7992b51#r66925255>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAAsZ07lgfSUfiXJ8MubDCSN38G1y7ozks5qLmFpgaJpZM4Iz2PM>
> .
>